ament_index_python/Quality_Declaration.md

```diff
@@ -99,7 +99,7 @@ New features are required to have tests before being added.
 
 ### Public API Testing [4.ii]
 
-As the API is not defined explicitly, this section is considered unresolved.
+All the functionality of this package is tested, having lines code coverage report result of [96%](https://ci.ros2.org/job/ci_linux_coverage/85/cobertura/ament_index_python/).
```

Comment:
Looking at the tests, I couldn't tell if each function was tested individually, or if they all were tested by the nature of the 'feature' tests. Do you know if each API function has associated tests?

Reply:
I can't tell for sure. But Public API Testing is considered as "Must have system, integration, and/or unit tests which cover all of the public API". So I think this shouldn't be a concern as long as the functions are covered.

Reply:
Ok cool. The sentence could use a little cleanup though.

```suggestion
All the functionality of the declared API in this package is covered in its unit tests. Currently it has a line coverage of [96%](https://ci.ros2.org/job/ci_linux_coverage/85/cobertura/ament_index_python/).
```